### PR TITLE
update base image to ubuntu 18.04

### PR DESCRIPTION
**Pull request by lizan**
Signed-off-by: Lizan Zhou <lizan@tetrate.io>

### CODE REVIEW

File: build_container/Dockerfile-ubuntu [hunk 1]
```diff
@@ -10,7 +10,7 @@ SHELL ["/bin/ash", "-o", "pipefail", "-c"]
 RUN curl -fsSL https://github.com/multiarch/qemu-user-static/releases/download/v${QEMUVERSION}/qemu-${QEMUARCH}-static.tar.gz | tar zxvf - -C /usr/bin
 RUN chmod +x /usr/bin/qemu-*
 
-FROM ${IMAGEARCH}ubuntu:xenial
+FROM ${IMAGEARCH}ubuntu:bionic
```
Comment (nickolaev) [inline build_container/Dockerfile-ubuntu:13]: My impression is that the builds for ppc65le and aarch64 are still dependent on xenial/16.04

Comment (lizan) [reply build_container/Dockerfile-ubuntu:13]: Not really, those tools built for xenial can run on bionic, it doesn't block base image update.

**Comment by lizan** [comment]
https://github.com/envoyproxy/envoy/issues/10944

**Pull request merged**